client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -130,9 +133,21 @@ export const useStoreItemInfo = ( {
 
 	const getIsIncludedInPlan = useCallback(
 		( item: SelectorProduct ) => {
+			// If user owns the Jetpack Complete Plan/bundle, then JetPack Security plan/bundle should
```

Comment:
Do you think adding Security to the list of Complete features would work?
https://github.com/Automattic/wp-calypso/blob/7513ef8a1f573aec8e36918b9cc9bad25cc29202/packages/calypso-products/src/plans-list.tsx#L1750

Reply:
This was a great idea! I tried testing it locally, though, and marking Security as included in Complete caused the Security card's call to action to change from "Get" (disabled) to "Manage Plan," which I think would be confusing to customers. 😞